base-rules/dell-redfish-v117.yaml

```diff
@@ -0,0 +1,104 @@
+  - Path: /redfish/v1/Chassis/{chassis}/ThermalSubsystem/Fans/{unit}
+  - Path: /redfish/v1/Chassis/{chassis}/ThermalSubsystem/ThermalMetrics
+
+  ## Dell
+  - Path: /redfish/v1/Dell/Chassis/{system}/DellAssembly/{assembly}
+  - Path: /redfish/v1/Dell/Systems/{system}/DellNumericSensor/{sensor}
+  - Path: /redfish/v1/Dell/Systems/{system}/DellPCIeFunction/{function}
+  - Path: /redfish/v1/Dell/Systems/{system}/DellPSNumericSensor/{sensor}
+  - Path: /redfish/v1/Dell/Systems/{system}/DellSensor/{sensor}
+  - Path: /redfish/v1/Dell/Systems/{system}/DellSlot/{slot}
+  - Path: /redfish/v1/Dell/Systems/{system}/Memory/DellMemory/{memory}
+  - Path: /redfish/v1/Dell/Systems/{system}/NetworkAdapters/NetworkDeviceFunctions/DellNICPortMetrics/{port}
+  - Path: /redfish/v1/Dell/Systems/{system}/NetworkDeviceFunctions/DellNIC/{nic}
+  - Path: /redfish/v1/Dell/Systems/{system}/NetworkDeviceFunctions/DellNICCapabilities/{nic}
+  - Path: /redfish/v1/Dell/Systems/{system}/NetworkPorts/DellSwitchConnection/{connection}
+  - Path: /redfish/v1/Dell/Systems/{system}/Processors/DellProcessor/{processor}
+  - Path: /redfish/v1/Dell/Systems/{system}/Storage/DellController/{controller}
+  - Path: /redfish/v1/Dell/Systems/{system}/Storage/DellPCIeSSD/{disk}
+  - Path: /redfish/v1/Dell/Systems/{system}/Storage/Drives/DellPhysicalDisk/{disk}
+  - Path: /redfish/v1/Dell/Systems/{system}/Storage/Volumes/DellVirtualDisk/{disk}
+  ## Fabrics
+  - Path: /redfish/v1/Fabrics/{fabric}
+  - Path: /redfish/v1/Fabrics/{fabric}/Switches
+  ## Managers
+  - Path: /redfish/v1/Managers/{manager}
+  - Path: /redfish/v1/Managers/{manager}/Attributes
+  - Path: /redfish/v1/Managers/{manager}/EthernetInterfaces/{interface}
+  - Path: /redfish/v1/Managers/{manager}/HostInterfaces/{host}
+  - Path: /redfish/v1/Managers/{manager}/NetworkProtocol
+  ## Systems
+  - Path: /redfish/v1/Systems/{system}
+  - Path: /redfish/v1/Systems/{system}/BootOptions/{option}
+  - Path: /redfish/v1/Systems/{system}/EthernetInterfaces/{interface}
+  - Path: /redfish/v1/Systems/{system}/Memory/{memory}
+  - Path: /redfish/v1/Systems/{system}/Memory/{memory}/MemoryMetrics
+  - Path: /redfish/v1/Systems/{system}/NetworkAdapters/{nic}
+  - Path: /redfish/v1/Systems/{system}/NetworkAdapters/{nic}/NetworkDeviceFunctions/{function}
+  - Path: /redfish/v1/Systems/{system}/NetworkAdapters/{nic}/NetworkPorts/{port}
+  - Path: /redfish/v1/Systems/{system}/NetworkInterfaces/{nic}
+  - Path: /redfish/v1/Systems/{system}/PCIeDevices/{device}
+  - Path: /redfish/v1/Systems/{system}/PCIeDevices/{device}/PCIeFunctions/{function}
+  - Path: /redfish/v1/Systems/{system}/Processors/{processor}
+  - Path: /redfish/v1/Systems/{system}/SimpleStorage/{controller}
+  - Path: /redfish/v1/Systems/{system}/Storage/{storage}
+  - Path: /redfish/v1/Systems/{system}/Storage/{storage}/Drives/{device}
+  - Path: /redfish/v1/Systems/{system}/Storage/{storage}/StorageControllers/{controller}
```

Comment:
If this is not used in v1.17, please remove.

Reply:
I follow your comment

Reply:
Both `StorageControllers` and`Controllers` seem to be active.